hooks/pre_gen_project.py

```diff
@@ -0,0 +1,6 @@
+{{ cookiecutter.update({"package_name": "inmanta-module-"+cookiecutter.module_name.lower().replace("_","-") }) }}
+{{ cookiecutter.update({"inmanta_module_name": (cookiecutter.module_name[0].lower()+cookiecutter.module_name[1:]).replace("-","_") }) }}
+
+print("===========================")
+print("Module name: {{ cookiecutter.inmanta_module_name }}")
+print("Python package name: {{ cookiecutter.package_name }}")
```

Comment:
After thinking about this some more I think I'm in favor of just converting the name silently, without this message, and to leave the git output as is. What do you think?

Reply:
I'm not particularly attached to those outputs :p I'll remove them.